src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -21,6 +21,10 @@
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
+#if !CODE_STYLE
+using OptionSet = Microsoft.CodeAnalysis.Options.OptionSet;
+#endif
```

Comment:
😕 Why is this one needed?

Reply:
I think all of these can be removed with the new APIs your added, but I wanted to avoid too much churn in this PR. I have left out just the OptionSet based `#if` directives in the PR, to be cleaned up in a follow-up PR.

Reply:
This file already has a using directive for `Microsoft.CodeAnalysis.Options`. Are there two definitions for `OptionSet`?